x-pack/plugins/apm/server/routes/services/get_services/merge_service_stats.test.ts

```diff
@@ -53,6 +53,12 @@ describe('mergeServiceStats', () => {
             serviceName: 'opbeans-java',
           },
         ],
+        alertCounts: [
```

Comment:
@yngrdyn In case a service has 2 environments and only one of the environments has an active alert.

What happens if I filter by the environment in the inventory page?

Reply:
AFAIK we don't want to discriminate per environment. @sqren should we get into account the environment here?

Reply:
++ iirc, the discussion we had was to stay agnostic of env (for the time being)